packages/core/documentation/Spacing/Spacing.tsx

```diff
@@ -42,8 +42,9 @@ const SpacingTableRow: React.FC<{ spacing: spacingClass }> = ({ spacing }) => {
             <td data-header="Spacing:" className="jkl-portal-spacing-example-table__data">
                 <div className={`jkl-${spacing}-top`} style={{ display: "none" }} ref={ref} />
                 <div
-                    aria-label={spacing}
+                    aria-label={spacing + ", " + pxValue}
```

Comment:
Veldig oldschool av deg da....

```suggestion
                    aria-label={`${spacing},  ${pxValue}`}
```

Reply:
Jeg ble litt nostalgisk :D